internal/cli/init.go

```diff
@@ -157,8 +163,32 @@ func (c *InitCommand) Run(args []string) int {
 
 	// If we have no config, initialize a new one.
 	if path == "" {
-		if !c.initNew() {
-			return 1
+		proceed, err := c.ui.Input(&terminal.Input{
+			Prompt: "Do you want help generating a waypoint.hcl file? Type 'yes' to initialize the interactive generator or 'no' to generate a template waypoint.hcl file: ",
+			Style:  "",
+			Secret: false,
+		})
+		if err != nil {
+			c.ui.Output(
+				"Error getting input: %s",
+				clierrors.Humanize(err),
+				terminal.WithErrorStyle(),
+			)
+		} else if strings.ToLower(proceed) == "yes" || strings.ToLower(proceed) == "y" {
+			c.ui.Output("Starting interactive .hcl generator.\n")
+			if !c.hclGen() {
+				return 1
+			}
+		} else if strings.ToLower(proceed) == "no" {
```

Comment:
Since we are allowing y/Y for "yes", can we allow n/N for "no" here?

Reply:
Actually, is this if/else necessary, if our else does the same thing (generate a template file)?

Reply:
Good point, I've gotten rid of the extraneous else if.